client/src/modules/shopping-list/components/CreationForm.jsx

```diff
@@ -57,6 +57,7 @@ class CreationForm extends Component {
 }
 
 CreationForm.propTypes = {
+  isShoppingList: PropTypes.bool,
```

Comment:
remove this - the form should not know anything about shopping lists or cohorts - it should be also common component